### ISSUE TITLE
Crash using VS Code with latest NVDA

### ISSUE THREAD

**Issue by chuntaojun** [Microsoft/vscode#23563]
- VSCode Version: Code - Insiders 1.11.0-insider (d98a518c5485bd7a47c70f5b3783f70efa5edcd6, 2017-03-29T06:11:14.271Z)
- OS Version: Windows_NT ia32 10.0.15063
- Extensions: none
---

Steps to Reproduce:

1.
2.

**Comment by aeschli** [comment]
Please provide more information

**Comment by chuntaojun** [comment]
Sorry.
I am a student of Chinese, yesterday I updated the NVIDIA graphics driver, before it can be opened using vscode, but in Bahrain after the display window fails, please restart the window to continue from the last stop position, and then I install the preview version of vscode, but shows expansion a host of unexpected termination. Please reload the window to restore"

![Uploading TIM图片20170329194055.jpg…]()

**Comment by chuntaojun** [comment]
![tim 20170329194055](https://cloud.githubusercontent.com/assets/23445172/24466174/7e5427d0-14e2-11e7-93ad-71e76d3b375d.jpg)

**Comment by chuntaojun** [comment]
Can you help me with this problem, thank you @aeschli

**Comment by bpasero** [comment]
If I understand correctly, updating to the latest NVDA and running VS Code caused a crash. I just tried this (getting an update for NVDA) and cannot reproduce the crash.

**Comment by chuntaojun** [comment]
thank you very much，I had solve this problem
